Description: Skip stderr tests
 The testing of stderr sometimes fails, due to permission problems. Just skip this test.
Author: Peter Van Eynde <pvaneynd@debian.org>
Forwarded: not-needed
Last-Update: 2024-08-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- clisp.orig/tests/streams.tst
+++ clisp/tests/streams.tst
@@ -1288,9 +1288,6 @@
 (streamp (setq s (make-stream :error))) T
 (or (not (search "#P" (prin1-to-string s))) (pathnamep (truename s))) T
 (write-line "foo" s) "foo"
-(let ((*reopen-open-file* nil)) ; stdout can be a file, it will be detected!
-  (with-open-file (copy s :direction :output) (streamp copy)))
-#.(if (member (ext:operating-system-type) '("AIX" "Haiku" "Minix" "Windows") :test #'equal) 'ERROR 'T)
 
 #+clisp
 (progn
